src/networks.json

```diff
@@ -1330,6 +1330,22 @@
     "start": 1683285,
     "logo": "ipfs://QmVh5XRi7xN4UNw7SNKpEhjF1TsPXJnRyNkB4EuMVTVCB6"
   },
+  "9052": {
+    "key": "9052",
+    "name": "Acrechain Mainnet",
+    "shortName": "Acrechain",
+    "chainId": 9052,
+    "network": "mainnet",
+    "multicall": "0x5B614BB4b2939dc4E5EC1399F16c7E78949764CB",
+    "rpc": [
+      "https://evm.acrescan.com"
+    ],
+    "explorer": {
+      "url": "https://acrescout.mindheartsoul.org"
+    },
+    "start": 1690100,
```

Comment:
```suggestion
    "start": 471283,
```

Reply:
Hello,

1. We created new Multicall contract and code is verified.
2. We have pinned the LOGO (IPFS)
3. Changed the start block to the block number where the new Multicall was created.